python/nav/web/business/reportengine.py

```diff
@@ -81,11 +82,17 @@ def build_context(period, report_type):
         ReportSubscription.LINK: utils.get_interface_records,
     }
     records = lookup[report_type](start, end)
+    sorted_records = sorted(records, key=lambda x: x.downtime, reverse=True)
+    max_length = 30
+    if records:
+        max_length = max([len(str(r.subject)) for r in records])
```

Comment:
you can safely drop the square brackets to make `max(len(str(r.subject)) for r in records)`